manifests/assisted-service/assisted-service-ingress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  namespace: assisted-installer
+  name: assisted-service
+  annotations:
+    cert-manager.io/issuer: assisted-installer-ca
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+spec:
+  rules:
+  - host: assisted-service-assisted-installer.apps.dev.redhat.com
+    http:
+      paths:
+      - pathType: Prefix
+        path: /
+        backend:
+          service:
+            name: assisted-service
+            port:
+              number: 8090
+  tls:
+  - hosts:
+    - assisted-image-service-assisted-installer.apps.dev.redhat.com
```

Comment:
The tls host name is different than the host name under the  rule?

Reply:
No, that is a mistake, I am fixing it.